speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -60,11 +60,14 @@ typedef unsigned int DataStorageType;
 #define DATA_STORAGE_TYPE_SQL       1
 
 /// @brief The different types of data that can be stored
+// TODO: update these to the newer database structure in the front-end.
+//  Current uses in the back-end for each value are listed below.
+//  Changing these without updating the front-end as well may break checkboxes and such
 typedef struct DataToStore
 {
-    bool EnvironmentData;
-    bool CarData;
-    bool HumanData;
-    bool InterventionData;
-    bool MetaData;
+    bool EnvironmentData; // Used as Gamestate
+    bool CarData; // Unused
+    bool HumanData; // Used as UserInput
+    bool InterventionData; // Used as Decisions
+    bool MetaData; // Unused
```

Comment:
The meaning of these cannot be changed as they are part of the requirements

Reply:
Comments have been moved to SQLDatabaseStorage now to properly mark where changes are required